build.gradle

```diff
@@ -60,21 +42,10 @@ subprojects {
   }
   
   dependencies {
-    implementation platform("org.springframework.boot:spring-boot-dependencies:$bootVersion")
-    testImplementation platform("org.junit:junit-bom:${spinnaker.version('jupiter')}")
+    implementation platform("com.netflix.spinnaker.kork:kork-bom:4.1.0-rc.3-springBoot2")
```

Comment:
Specify the kork-bom as a platform dependency (I was already using the Spring Boot and JUnit ones but they are pulled in transitively by `kork-bom` now.